python/lsst/sims/photUtils/Sed.py

```diff
@@ -995,23 +1022,26 @@ def calcNonSourceNoiseSq(self, skySed, hardwarebandpass, readnoise, darkcurrent,
         #This method outputs all of the parameters calculated along the way
         #so that the verbose version of calcSNR_psf still works
 
+        if photParams is None:
+            raise RuntimeError("Need to pass an instantiation of PhotometricParameters to " +
+                               "calcNonSourceNoiseSq")
+
         #Calculate the effective number of pixels for double-Gaussian PSF
-        neff = self.calcNeff(seeing, platescale)
+        neff = self.calcNeff(seeing, photParams.platescale)
```

Comment:
I think I'd rather just remove the default seeing values.  I am worried about users forgetting to set the seeing kwarg and then accidentally using LSST default values when they did not mean to.